opam

```diff
@@ -20,7 +24,7 @@ depends: [
   "rpc"
   "fd-send-recv"
   "uuidm"
-  "xapi-stdext" {>= "2.0.0"}
```

Comment:
Can't say whether this is correct but I'm not sure accumulating constraints is good in the long run. I would tend to add them as needed. There are two typical scenarios: the outside world has marched on and we need to add x `<=` because we are not yet ready for it. Or we have marched on and must demand `>=` because we rely on new features.